go/sample-apps/function/go.sum

```diff
@@ -1,613 +1,7 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
```

Comment:
This change seems unrelated to the PR

Reply:
hi @serkan-ozal updated the PR. created new [pr](https://github.com/open-telemetry/opentelemetry-lambda/pull/1645) for go changes